cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/KafkaCruiseControlServlet.java

```diff
@@ -799,7 +807,7 @@ private String generateGoalAndClusterStatusAfterExecution(boolean json, GoalOpti
     return sb.toString();
   }
 
-  private boolean addOrRemoveBroker(HttpServletRequest request, HttpServletResponse response, EndPoint endPoint)
+  private boolean addRemoveOrFixBroker(HttpServletRequest request, HttpServletResponse response, EndPoint endPoint)
```

Comment:
in parameter checking part of this method, the `throttleAddedOrRemovedBrokers()` does assume it is either a add_broker or remove_broker request.